en/manage-backup.md

```diff
@@ -0,0 +1,114 @@
+Title: Backup
+table_of_contents: off
+
+# Backup
+
+MAAS doesn't yet include specific tools to help with the backup and restoration
+of a working MAAS environment. As MAAS servers are obviously no different to
+the majority of other Linux-based servers, it's likely your current backup and
+disaster recovery solution will already include your MAAS environment. 
+
+However, as it's important to know which elements of MAAS are critical when
+restoring a previous deployment, we're going to briefly outline the process and
+its requirements below.
+
+## Configuration files
+
+The following MAAS components on each region and rack controller need to be
+backed-up and restored to recreate a working environment:
+
+1. The Postgresql database
+1. The configuration files in `/etc/maas`
+1. The configuration files in `/var/lib/maas`
+
+`/var/lib/maas/boot-resources` can safely be excluded as this contains images
+easily re-downloaded within MAAS. Similarly, the remainder of `/var/lib/maas`
+can be ignored too in most cases, especially when the region and rack controllers
```

Comment:
"in most cases" - These phrasings should be avoided as they just make people wonder whether *their* case is OK or not.
"especially when" - So does the user need to do something if regiond and rackd are not on the same machine? If not, better not mention it at all.